components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -144,6 +148,28 @@ const ShouldThrottleFetchNewClassIdsRules = (): boolean => {
 }
 
 const fetchNewClassIdRules = () => {
+  let i = notYetQueriedElements.length
+  while (i > 0) {
+    i -= 1
```

Comment:
If we don't implement any limits, why we use the backward order, not just `for (.. of ..)`?